src/index.ts

```diff
@@ -134,26 +193,43 @@ export interface IOrizuruMessage<C extends Orizuru.Context, M> {
 	 * The message.
 	 */
 	message: M;
+
+}
+
+/**
+ * The server implementation.
+ * By default, we use [Express](https://expressjs.com/).
+ * This allows us to easily set up a mock implementation for testing.
```

Comment:
Is this comment is aimed at developers of this repo?
It should probably be aimed at the consumer of the library, for whom this would be a red herring.